Add privileged options to enable ip forwarding

AlMaVizca opened the pull request.

AlMaVizca committed:
Add privileged options to enable ip forwarding

Signed-off-by: Aldo Maria Vizcaino <aldo.vizcaino87@gmail.com>

suda commented:
Thanks Aldo! Looks good! Can you update the docs by running `.github/helm-docs.sh` and committing the changes to fix the linter error?

suda commented:
Shoot, looks like there are some more linting errors, could you investigate it?

AlMaVizca committed:
Remove unexpected element source

https://github.com/suda/charts/actions/runs/9792028093/job/27041946389?pr=15#step:4:79

AlMaVizca commented:
I've tested the linting on a [local branch](https://github.com/AlMaVizca/helm-charts/actions/runs/10028749431/job/27716089526), and it seems to be passing, but it's failing on testing.
I've built on my fork main branch the versions that I'm proposing on these PRs, and I'm not sure if it might be related to that.
Do you have any other ideas to investigate?
